torch/csrc/jit/script/parser.h

```diff
@@ -280,11 +261,13 @@ struct Parser {
       case TK_GLOBAL: {
         auto range = L.next().range;
         auto idents = parseList(TK_NOTHING, ',', TK_NOTHING, &Parser::parseIdent);
+        expectEndOfLine();
```

Comment:
Was this a bug?

Reply:
It was. I can't remember exactly when it failed but it was something like `return 2a+1` (this would parse correctly as a return and then an add)